File: test/benchmark/fib.rb
def fib(n)
  if n < 2 then
    n
  else
    fib(n - 1) + fib(n - 2)
  end
end

start = Time.now
for i in 0...5
  puts fib(28)
end
puts "elapsed: " + (Time.now - start).to_s

File: test/benchmark/fib.py
from __future__ import print_function

import time

def fib(n):
  if n < 2: return n
  return fib(n - 1) + fib(n - 2)

start = time.process_time()
for i in range(0, 5):
  print(fib(28))
print("elapsed: " + str(time.process_time() - start))